gnupg-pkcs11-scd/command.c

```diff
@@ -1376,10 +1376,15 @@ gpg_error_t _cmd_pksign_type (assuan_context_t ctx, char *line, int typehint)
 	}
 
 	if (need_free__data) {
```

Comment:
I moved `need_free__data = 1;` to after the malloc, thanks!

Reply:
Moving it to after the `malloc()` may miss it if the second `malloc()` (for `_data->data`) fails.

Reply:
yes, it will be `free(NULL)`